### ISSUE TITLE
Character problem - stucked in a phase

### ISSUE THREAD

**Issue by Annecleare** [dalaranwow/dalaran-wow#4820]
Hello,
I've got a level 80 Bloodelf Death Knight, Anneclaire
I was doing http://www.wowhead.com/quest=13236/army-of-the-damned quest when I had a disconnection from the server and I think I was stucked in the wrong phase.
I don't have any buff on me (or I can't see it). I can't see any NPCs/Players. I can't interact with anything.
I was logged in with my other character on the same account and there wasn't any issues.
I would be happy if anyone have a look on it! 
Many thanks

[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Annecleare** [comment]
Still waiting for response...  is there any GM used to be online ingame?
Sorry guys, but it's happened almost a week before.... and it's not just a general bug report. I can not play.
So I still waiting for someone to fix it, pretty please :)))

**Comment by DragonzMate** [comment]
I also have experienced this same error on my level 79 orc Death Knight and I attempted to resolve by going back to Dalaran but I can no longer see any NPC

**Comment by Annecleare** [comment]
Hi there,
Go back to Icecrown, wheres the quest is. You gonna find some NPC-s.
Aattack them and let them to kill you. Than u gonna get back to the right
phase.
